### PR TITLE
feat: feature to see altitude from pressure added in Barometer

### LINKED ISSUE #1598: Feature Request: Altitude Meter
**Actual Behaviour**
Currently no altitude meter

**Expected Behaviour**
Since there is barometer, we can calculate the altitude for those devices which supports barometer
**Would you like to work on the issue?**
Yes

@harsh-2711  @CloudyPadmal  @cweitat kindly share your view on this feature

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1598 

**Changes**: feature to see altitude from pressure added in Barometer

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[altitude.zip](https://github.com/fossasia/pslab-android/files/3498450/altitude.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java [hunk 1]
```diff
@@ -564,4 +612,12 @@ private void initiateBaroSensor(int type) {
                 break;
         }
     }
+
+    private float getAltitude(float pressure) {
+        if (pressure <= 0.0) {
+            return  0;
+        } else {
+            return (float) (44330 * (1 - Math.pow(pressure, 1.0 / 5.255)));
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java]: Where did you get these values and equations? Better mention reference as a comment

Comment (neel1998) [reply app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java]: https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf check section 3.6 on page 16

File: app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java [hunk 2]
```diff
@@ -372,8 +403,11 @@ private void setupInstruments() {
         y.setDrawGridLines(true);
         y.setLabelCount(10);
 
-        y2.setDrawGridLines(false);
-        y2.setMaxWidth(0);
+        y2.setTextColor(Color.WHITE);
+        y2.setAxisMinimum(0);
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java]: Implement a similar approach like 401 and 402 without hardcoded ranges.

**Comment by neel1998** [comment]
@CloudyPadmal please review

**Pull request merged**